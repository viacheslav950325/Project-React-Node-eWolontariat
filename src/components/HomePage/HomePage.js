import { Box } from "@material-ui/core";

const HomePage = () => {
    return(
        <Box
            display={"flex:"}
            height = {"100%"}
            justifyContent={"center"}
            alignItems={"center"}
            >
                <h1>Hello</h1>
        </Box>
    )
}

export default HomePage;