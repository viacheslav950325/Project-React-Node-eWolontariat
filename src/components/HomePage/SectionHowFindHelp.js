import { Box, Typography,Button } from "@material-ui/core";
import PlanningImg from '../../assets/img/planning.svg';

const SectionHowFindHelp = () => {
    return(
        <Box id={"section-how-find-help"}
            display={"flex:"}
            height = {"100%"}
            justifyContent={"center"}
            alignItems={"center"}
        >
            <Typography variant="h1">Jak znaleźć pomoc na pomocny.pl</Typography>
            <Typography variant="body1" paragraph={true}>
                    Pomocny.pl to nie tylko strona z ogłoszeniami dotyczącymi wolontariatu. To miejsce spotkań ludzi, którzy potrzebują przestrzeni 
                    na rozmowę o tym jak można im pomóc. Jeżeli jesteś w trudnej sytuacji i jest coś, w czym można Cię wesprzeć, możesz stworzyć nowe 
                    zadanie lub odezwać się bezpośrednio do któregoś z wolontariuszy. 
                </Typography>
            
            <Box display={'flex'} padding={'4rem'} justifyContent={'space-evenly'} mr={'2rem'}> 
                <Box component="img"  maxHeight={'400px'} alt="" src={PlanningImg}/>
                <Box alignSelf={'center'}>
                    <Typography variant={'h3'}>Krok 1.</Typography>
                    <Typography variant={'subtitle1'} paragraph={true}>
                        Aby rozpocząć kliknij przycisk Stwórz zadanie.
                    </Typography>
                    <Button variant="contained" size={'medium'} color='primary' href={"/"}>Stwórz zadanie</Button>
                    <Typography variant={'h3'}>Krok 2.</Typography>
                    <Typography variant={'subtitle1'} paragraph={true}>
                        Zostaniesz przekierowany do formularza, który poprowadzi Cię dalej. W odpowiednich miejscach wpisz najważniejsze informacje 
                        dotyczące tworzonego przez Ciebie zadania: krótki opis tego, czego potrzebujesz oraz ilu wolontariuszy może się w to zaangażować. 
                        Możesz dodać również zdjęcie.
                    </Typography>
                    <Typography variant={'h3'}>Krok 3.</Typography>
                    <Typography variant={'subtitle1'} paragraph={true}>
                        Kiedy wolontariusz zgłosi się do Twojej akcji, zostaniesz o tym powiadomion_ mailowo.
                    </Typography>
                </Box>
            </Box>
                
        </Box>
    )
}

export default SectionHowFindHelp;