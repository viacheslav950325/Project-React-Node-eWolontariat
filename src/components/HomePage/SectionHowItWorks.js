import { Box, Button, Typography,Card, CardMedia, CardActionArea, CardContent, CardActions} from "@material-ui/core";
import ImgFindCard from '../../assets/img/find-volunteer.svg';
import ImgBeVolunteer from '../../assets/img/be-volunteer.svg';

const SectionHowItWorks = () => {
    return(
        <Box id={"section-how-it-works"}
            display={"flex:"}
            height = {"100%"}
            justifyContent={"center"}
            alignItems={"center"}
        >
            <Typography variant="h1">Zobacz jak to działa</Typography>
            <Box display={'flex'} padding={'4rem'} justifyContent={'space-evenly'} mr={'2rem'}>
                <Card>
                        <CardMedia 
                            component={'img'}
                            alt="Grafika obrazująca szukanie w Internecie"
                            maxHeight="200"
                            image={ImgFindCard}
                            title="Znajdź wolontariusza">
                        </CardMedia>
                        <CardContent>
                        <Typography gutterBottom variant="h5" component="h2">
                            Znajdź wolontariusza
                        </Typography>
                        <Typography variant="body2" color="textSecondary" component="p">
                            Napisz czego Ci potrzeba i stwórz zadanie
                        </Typography>
                        </CardContent>
                    <CardActions>
                        <Button size="medium" variant="contained" color="primary">
                            Stwórz zadanie
                        </Button>
                    </CardActions>
                </Card>
                <Card>
                        <CardMedia 
                            component={'img'}
                            alt="Grafika obrazująca szukanie w Internecie"
                            maxHeight="200"
                            image={ImgBeVolunteer}
                            title="Znajdź wolontariusza">
                        </CardMedia>
                        <CardContent>
                        <Typography gutterBottom variant="h5" component="h2">
                            Znajdź wolontariusza
                        </Typography>
                        <Typography variant="body2" color="textSecondary" component="p">
                            Napisz czego Ci potrzeba i stwórz zadanie
                        </Typography>
                        </CardContent>
                    <CardActions>
                        <Button size="medium" variant="contained" color="success">
                            Załóż profil
                        </Button>
                    </CardActions>
                </Card>
            </Box>
        </Box>
    )
}

export default SectionHowItWorks;