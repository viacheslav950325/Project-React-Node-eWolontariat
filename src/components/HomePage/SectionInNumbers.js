import { Box, Button,Typography } from "@material-ui/core";
import LogoPomocny from "../../assets/img/logo-pomocny.svg";
import ImgGlobe from "../../assets/img/globe-mask.svg";

const SectionInNumbers = () => {
    return(
        <Box id={"section-in-numbers"}
            display={"flex:"}
            height = {"100%"}
            justifyContent={"center"}
            alignItems={"center"}
        >
            <Box sx= {{ display: 'flex', flexDirection: 'column', alignItems: "center"}} >
                <Box component="img" height={'60px'} alt="Logo pomocny.pl" src={LogoPomocny}/>
                <Typography variant="h2">w liczbach</Typography>
            </Box>
            <Box display={'flex'} padding={'4rem'} justifyContent={'space-evenly'} mr={'2rem'}>
                <Box alignSelf={'center'} textAlign={'center'}>
                    <Typography  className={"bigNumber"}>100</Typography>
                    <Typography variant="caption">Tylu osobom pomogliśmy</Typography>
                </Box>
                <Box alignSelf={'center'} textAlign={'center'}>
                    <Typography  className={"bigNumber"}>12334</Typography>
                    <Typography variant="caption">Tyle zadań zakończyło się sukcesem</Typography>
                </Box>
                <Box alignSelf={'center'} textAlign={'center'}>
                    <Typography  className={"bigNumber"}>345</Typography>
                    <Typography variant="caption">Tyle zgłosiło się wolontariuszy</Typography>
                </Box>
                
            </Box>
            <Box display={'flex'} padding={'4rem'} justifyContent={'space-evenly'} mr={'2rem'}>
                <Box alignSelf={'center'}>
                    <Typography variant={'body1'} paragraph={true}>
                        Imponujące? Dołącz do nas już dziś i bądź pomocny!
                    </Typography>
                    <Button variant="contained" color='success' href={"/"}>Zostań wolontariuszem</Button>

                </Box>
                <Box component="img" maxWidth={'400px'} alt="" src={ImgGlobe}/>
            </Box>
        </Box>
    )
}

export default SectionInNumbers;