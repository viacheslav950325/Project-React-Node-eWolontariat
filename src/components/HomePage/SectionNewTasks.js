import { Box, Typography} from "@material-ui/core";


const SectionNewTasks = () => {
    return(
        <Box id={"section-new-tasks"}
            display={"flex:"}
            height = {"100%"}
            justifyContent={"center"}
            alignItems={"center"}
        >
            <Typography variant="h1">Zobacz jakiej pomocy potrzebujÄ… inni</Typography>
            <Box display={'flex'} padding={'4rem'} justifyContent={'space-evenly'} mr={'2rem'}>
               
               
            </Box>
        </Box>
    )
}

export default SectionNewTasks;