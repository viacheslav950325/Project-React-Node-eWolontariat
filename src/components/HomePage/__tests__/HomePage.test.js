import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import HomePage from "../HomePage";

describe("HomePage",()=>{

    it("Should render TasksPage correctly",()=>{
        const wrapper = shallow(<HomePage/>);
        expect(toJson(wrapper)).toMatchSnapshot();
    });

});