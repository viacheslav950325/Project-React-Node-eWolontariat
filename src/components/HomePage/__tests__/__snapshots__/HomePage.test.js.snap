// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Should render TasksPage correctly 1`] = `
<Fragment>
  <SectionHero />
  <SectionHowItWorks />
  <SectionNewTasks />
  <SectionTheBestVolunteers />
  <SectionHowFindHelp />
  <SectionInNumbers />
</Fragment>
`;
